substrate/frame/state-trie-migration/src/lib.rs

```diff
@@ -79,7 +79,13 @@ pub mod pallet {
 		dispatch::{DispatchErrorWithPostInfo, PostDispatchInfo},
 		ensure,
 		pallet_prelude::*,
-		traits::{Currency, Get},
+		traits::{
+			fungible::{
+				hold::Balanced as FunBalanced, Inspect as FunInspect,
```

Comment:
I don't think we even need `as` here because only `fungible` is used in this scope.

If we have conflicts in a single scope, my preference is to still avoid `as` and just use the existing namespacing that comes with our crate structure e.g. write `fungible::Balanced` and `fungibles::Balanced`.

Reply:
@0xmovses yes, agree, those `Fun`/`Fn` stuff is no good, I've raised this also few days ago [see the discussion bellow](https://github.com/paritytech/polkadot-sdk/issues/226#issuecomment-1822800585)

so I will try to refactor this everywhere (a separate PR) as we discussed this with Liam already here: https://github.com/paritytech/polkadot-sdk/issues/226#issuecomment-1823007235

Reply:
Oh great, yes I've seen this naming scope issue everywhere. I think the approach agreed upon in the issue is the nicest one.